revisions/README_PacBio_CCS_10kb_HG002.md

```diff
@@ -0,0 +1,25 @@
+README - High-fidelity long read dataset of HG002, Ashkenazim Son
+Last Updated:  August 31, 2018 
+
+BACKGROUND
+This README file contains information for the SMRT Sequencing data from a 
+HG002/NA24385/huAA53E0, Ashkenazim Son sample using Binding/Sequencing chemistry 
+3.0 and SMRT Link 6.0. One shotgun library was prepared from a NIST reference material
```

Comment:
change language to library was prepared using genomic DNA for the HG002 GIAB sample. DNA was sourced from NIST reference material (RM8392, ...